app/ocr.py

```diff
@@ -189,22 +206,28 @@ def get_date(self) -> str:
         return '{}-{}-{}'.format(sorted_dates[0].year, sorted_dates[0].month, sorted_dates[0].day)
 
     def get_panel(self) -> str:
-        """
-        • Returns the panel members of case in document.
-        """
-        panel_members: List[str]
-        panel_members = []
-        possible_members: Iterator[Span]
-        possible_members = map(
-            lambda ent: ent.text, self.get_ents(['PERSON'])
-        )
-        for member in possible_members:
-            judge: Union[str, None]
-            judge = self.if_judge(member)
-            if judge:
-                panel_members.append(judge)
-
-        return '; '.join(set(panel_members))
+         """
+         • Returns a semicolon separated string of panel members for appellate case documents.
+         """
+
+         matcher = PhraseMatcher(nlp.vocab) # Create the phrase matcher
+         uid = 0 # Unique identifier for each judge's patterns
+
+         # Add two patterns to the matcher for each judge
+         for judge in panel_members:
+             matcher.add(f'PATTERN_{uid}', [nlp(judge)]) # Just the name as it is in the list
+             matcher.add(f'PATTERNX_{uid}', [nlp(judge.replace(".",""))]) # A pattern that looks for the names without periods, sometimes not picked up by the scan
+             uid += 1
+
+         matches = matcher(self.doc) # Run the phrase matcher over the doc
+         case_panel_members = set()
+
+         if len(matches) > 0: # If a judge from our list is found in the document
+             for match_id, start, end in matches:
+                 judge = self.doc[start:end] # A span object of the match, fetched with its `start` and `end` indecies within the doc
+                 case_panel_members.add(judge.text) # Extract the text from the span object, and add to the set of panel members
+
+         return '; '.join(case_panel_members)

```

Comment:
Output the panel members as a list here.
```suggestion
         return list(case_panel_members)
```

Reply:
Yes, I actually remember having a conversation about this, where I believe I am supposed to return a list, so thank you for pointing this out!